packages/terra-tabs/src/Tabs.jsx

```diff
@@ -84,6 +84,7 @@ class Tabs extends React.Component {
     this.state = {
       activeKey: this.getInitialState(),
       isLabelTruncated: false,
+      showCollapsedTabs: undefined,
```

Comment:
Should this be false?

Reply:
Updated here https://github.com/cerner/terra-framework/pull/972/commits/e2424685bd0d35297cd4932e47e57f1b0032bedd.